musit-service/src/main/scala/no/uio/musit/service/NoCacheFilter.scala

```diff
@@ -0,0 +1,26 @@
+package no.uio.musit.service
+
+import javax.inject.Inject
+
+import akka.stream.Materializer
+import play.api.http.{HeaderNames, Status}
+import play.api.mvc._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class NoCacheFilter @Inject() (
+    implicit val mat: Materializer,
+    ec: ExecutionContext
```

Comment:
Remove `ec: ExecutionContext` since it's imported implicitly using `import play.api.libs.concurrent.Execution.Implicits.defaultContext`.